packages/meetup-oauth/meetup_server.js

```diff
@@ -10,6 +10,12 @@ OAuth.registerService('meetup', 2, null, query => {
   return {
     serviceData: {
       id: identity.id,
+      name: identity.name,
+      lang: identity.lang,
+      link: identity.link,
+      photo: identity.photo,
+      county: identity.country,
```

Comment:
Shouldn't this field be named country ? :slightly_smiling_face:

Reply:
Ah, yes! Thanks!